config.rb

```diff
@@ -85,4 +87,17 @@
 after_build do |builder|
   builder.source_paths << File.dirname(__FILE__)
   builder.copy_file('data/_redirects', 'build/_redirects')
+
+  page_twos = sitemap.resources.select { |resource| resource.url =~ /\/page-2\/$/ }
+  puts page_twos.map &:url
+
+  # The pagination doesn't generate /page-1/ urls, so put in redirects
+  # for them that take you back to the "parent" (which contains the 1st page)
+  builder.append_to_file('build/_redirects') do
+    page_twos.map do |page_two|
+      parent_url = page_two.url.gsub(/\/page-2\/$/, '/')
+      page_one_url = page_two.url.gsub(/\/page-2\/$/, '/page-1/')
+      "#{page_one_url}  #{parent_url}"
+    end.join("\n")
+  end
```

Comment:
Yeah, there's no way to generate the url, but curious folk might try to hack the url, and it's something we can mitigate against so why not.  We already do it for some dangling path segments (e.g. we have `blog/author` -> `blog` to avoid giving 404s to url hackers).